bloom-housing/bloom
# Disables listing status cron

# Pull Request Template

## Issue Overview

## Description

This disables the listing status cron job, since Detroit doesn't currently utilize the application process. If we re-enable it for Detroit, we should refactor it, since it currently takes about 1.2 seconds on average to run. If the system has thousands of listings, then that time will increase.

## How Can This Be Tested/Reviewed?

This is hard to test locally, without having it up for an hour. Once it's on dev though, we should see the periodic run of this drop.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
